### ISSUE TITLE
#modxbughunt moving imageTVs to first tab - Width issues

### ISSUE THREAD

**Issue by pepebe** [modxcms/revolution#13326]
### Summary
If you are moving imageTVs to

modx-resource-main-left
or
modx-resource-main-right

causes all kind of layout issues on smaller screens:

![screenshot 2017-03-03 12 57 04](https://cloud.githubusercontent.com/assets/667315/23550426/0c832b82-0011-11e7-9711-666273d89dee.png)
![screenshot 2017-03-03 12 56 44](https://cloud.githubusercontent.com/assets/667315/23550425/0c82cdfe-0011-11e7-9724-ae0e81b7251b.png)

### Step to reproduce
Use form customisation to move an imageTv to either

modx-resource-main-left
or
modx-resource-main-right

### Observed behavior
image TVs start to break the right column very fast. At a certain point imageTVs on the left side overlap elements in the right column.

### Expected behavior
Max. width should be the width of the surrounding column. A fixed width (360px) can't be contained by a container and therefore the tv is breaking things.

### Environment
Any.

**Comment by alroniks** [comment]
The left region works fine, but the right region still has issues.
![Editing_ Home _ MODX Revolution](https://user-images.githubusercontent.com/303498/113624536-73e3b480-9668-11eb-8604-510d9bf83728.png)

@Ruslan-Aleev Could you have a look, please?

**Comment by Ruslan-Aleev** [comment]
@alroniks Yes, I tried to edit the width of the TV (image, file), make it in %, but the question is not clear.
1) Global styles cannot be set, because they will be overridden by the styles of the TV component.
At the same time, the styles inside the component feel like they don't work, you can change them, but there will be no effect, I'm talking about:
https://github.com/modxcms/revolution/blob/1c413a082b139c6806eff6fd422436deccc78c8a/manager/templates/default/element/tv/renders/input/image.tpl#L38
Those, it is not clear where the width is set, in the end.

2) Even if it turns out to set the TV width in %, how will it look? I thought about it, see https://github.com/modxcms/revolution/issues/11856#issuecomment-453766597
